composer.json

```
@@ -3,15 +3,15 @@
     "description": "Shell for PHPCR",
     "require": {
         "php": "^8.0",
-        "symfony/console": "^5.0 || ^6.0",
+        "symfony/console": "^5.0 || ^6.0 || ^7.0",
         "jackalope/jackalope": "^1.3.4",
         "phpcr/phpcr": "^2.1",
-        "phpcr/phpcr-utils": "^1.2",
-        "symfony/finder": "^5.0 || ^6.0",
-        "symfony/serializer": "^5.0 || ^6.0",
-        "symfony/yaml": "^5.0 || ^6.0",
-        "symfony/dependency-injection": "^5.0 || ^6.0",
-        "symfony/expression-language": "^5.0 || ^6.0",
+        "phpcr/phpcr-utils": "^1.2 || ^2.0",
```

can we try the dev version for now, to avoid a broken release of phpcr-utils?
```suggestion
        "phpcr/phpcr-utils": "^1.2 || ^2.0@dev",
```

---

Did apply this in the CI to test against `@dev` dependencies. We still are blocked by phpspec and behat here to test against Symfony 7:

 - https://github.com/phpspec/phpspec/pull/1452
 - https://github.com/Behat/Behat/pull/1442